src/extractor/CaretValueRuleExtractor.ts

```diff
@@ -199,16 +199,19 @@ export class CaretValueRuleExtractor {
   static processResource(
     input: any,
     fisher: utils.Fishable,
-    resourceType: 'ValueSet' | 'CodeSystem',
-    config: fshtypes.Configuration
+    resourceType: 'ValueSet' | 'CodeSystem' | 'Concept',
```

Comment:
I don't think we want to do this, the function is `processResource`, and the param is `resourceType`, so it feels off to accept a `Concept` here, since Concepts are not a type of FHIR resource.

Reply:
And you end up having to do a solid amount of special casing for `Concept` anyway, so I think it's probably best to just make this its own thing.